defaults/docs-defaults.yaml

```diff
@@ -285,13 +294,13 @@ defaults:
     web: impression
   default_video_player_size_bytes: 192205
   generic_creative_ad_server:
-    emissions_per_creative_request_per_geo_gco2pm:
```

Comment:
can you keep the old names as well for now, please?

Reply:
good catch, that was a problem in the file generation, should be fine now